spec/system/candidate_interface/decoupled_references/candidate_requests_a_reference_spec.rb

```diff
@@ -43,10 +43,17 @@
     then_i_am_told_a_reference_has_already_been_sent
 
     when_i_have_added_an_incomplete_reference
-    and_i_visit_the_reference_review_page
+    and_i_visit_the_all_references_review_page
     then_i_should_not_see_a_send_reference_link
     when_i_navigate_to_the_send_request_page
     then_i_should_see_a_not_found_message
+
+    when_i_have_a_cancelled_reference
+    and_i_visit_the_all_references_review_page
+    and_i_click_the_resend_reference_link
+    and_i_confirm_that_i_am_ready_to_send_a_reference_request
+    then_i_see_a_confirmation_message
+    and_the_reference_is_moved_to_the_requested_state
```

Comment:
is it worth including an assertion about the email that's sent as well?

Reply:
Yes should have added that. Done now.